src/modules/pokemons/PokemonList.ts

```diff
@@ -21192,10 +21193,10 @@ export const pokemonList = createPokemonArray(
         'exp': 56,
         'catchRate': 190,
         'evolutions': [
-            TimeRestrictedLevelEvolution(6, 17, 'Rockruff', 'Lycanroc (Midday)', 25),
-            TimeRestrictedLevelEvolution(17, 18, 'Rockruff', 'Lycanroc (Dusk)', 25),
-            TimeRestrictedLevelEvolution(18, 5, 'Rockruff', 'Lycanroc (Midnight)', 25),
-            TimeRestrictedLevelEvolution(5, 6, 'Rockruff', 'Lycanroc (Dusk)', 25),
+            TimeRestrictedLevelEvolution(DayCyclePart.Day, true, 'Rockruff', 'Lycanroc (Midday)', 25),
+            TimeRestrictedLevelEvolution(DayCyclePart.Night, true, 'Rockruff', 'Lycanroc (Midnight)', 25),
+            TimeRestrictedLevelEvolution(DayCyclePart.Dusk, true, 'Rockruff', 'Lycanroc (Dusk)', 25),
+            TimeRestrictedLevelEvolution(DayCyclePart.Dawn, true, 'Rockruff', 'Lycanroc (Dusk)', 25),
```

Comment:
this changes the hours from 5-6 to 6-5. Might not be wanted, dawn isn't used anywhere else so we could move it to 5 AM (instead of 6). this way dawn is part of the night and Dusk is part of the day. Which would make this (to get current, replace dawn but : 6 -7)

Dawn : 5 - 6
Day:  6 - 18
Dusk: 17 - 18
Night: 18 - 5